tests/common/functions

```diff
@@ -256,7 +256,7 @@ run_mpi_binary () {
     input_file=""
     output_file=""
     if [ "$RESOURCE_MANAGER" = "slurm" ];then
-	timeout=2		# default job timeout (in minutes)
+	timeout=5		# default job timeout (in minutes)
```

Comment:
I think we should only change the timeout for Github Actions and Cirrus CI runs. Maybe make this dependent on the SIMPLE_CI variable. I do not want to change the behaviour of the test suite in Jenkins.

Reply:
Done with https://github.com/openhpc/ohpc/pull/1703/commits/c2b6c1f1b420ab68f3fa25fdd085c4034ec88816